specification/storagepool/resource-manager/Microsoft.StoragePool/preview/2020-03-15-preview/storagepool.json

```diff
@@ -805,6 +907,56 @@
             "create",
             "read"
           ]
+        },
+        "additionalCapabilities": {
```

Comment:
Please add it to sample of the API

Reply:
Discussed offline -- we're intentionally leaving this field a little obscure